docs/rfcs/002-archival_storage/README.md

```diff
@@ -341,6 +348,44 @@ b24923333/
       Tag-File-Character-Encoding: UTF-8
 ```
 
+#### Partial bag
+
+Note that all files must be present the manifest and only files that are not supplied present in `fetch.txt`.
+
+```
+b24923333/
+|-- data
+|   |-- b24923333.xml      // mets "anchor" file for multiple manifestation
+|  [|-- b24923333_001.xml  // mets file for vol 1]
+|  [|-- b24923333_002.xml  // mets file for vol 2]
+|   ...
```

Comment:
I know these examples have omissions, but for clarity can these files be the same as those specified in the tag file examples?

Reply:
Is it better now?

Reply:
👍 yes exactly that, thanks.